app/grants/migrations/0005_grant_slug.py

```diff
@@ -24,6 +24,7 @@ class Migration(migrations.Migration):
             model_name='grant',
             name='slug',
             field=django_extensions.db.fields.AutoSlugField(blank=True, editable=False, populate_from='title'),
-        ),
-        migrations.RunPython(update_grants),
+        )
+        # ,
+        # migrations.RunPython(update_grants),
```

Comment:
this one too

Reply:
But this is the point at which I was receiving the error last week and you said to just comment out the offending line so we could move forward

Reply:
@SaptakS did you run into this issue when you rebased?

Reply:
I ran into there issues again this morning when I pulled down the lastest `grants`

Reply:
Since I already have eveything migrated there are no errors while running `make migrations` or `make migrate`. I can uncomment the line.